api/managementpb/dbaas/components.proto

```diff
@@ -58,6 +58,32 @@ message GetPXCComponentsResponse {
   repeated Version versions = 1;
 }
 
+// SetComponent contains fields to manage components.
+message SetComponent {
+  string default_version = 1;
+  repeated string disabled_versions = 2;
+}
+
+message SetPSMDBComponentsRequest {
+  // Kubernetes cluster name.
+  string kubernetes_cluster_name = 1;
+  SetComponent mongod = 2;
+  SetComponent backup = 3;
+}
+
+message SetPSMDBComponentsResponse {}
+
+message SetPXCComponentsRequest {
```

Comment:
Shouldn't we allow to pass pmm here too?

Reply:
Good point. But version server contains outdated pmm client. I'll create a ticket for it.

Reply:
https://jira.percona.com/browse/PMM-7814

Reply:
Got it!! 👍